manifests/selector.pp

```diff
@@ -6,6 +6,7 @@
   $rewrite = undef,
   $newurl = undef,
   $movedto = undef,
+  $order = 10,
```

Comment:
No reason, given that it currently isn't possible for the order to be modified, the default value is arbitrary,